README.md

```diff
@@ -1,22 +1,55 @@
 # Robotics Essentials ROS 2
 
 Welcome to an open-source repository that contains materials and exercises needed 
-to learn essential robotics skills using ROS 2. 
+to learn the essential robotics skills using ROS 2. 
 
 This material is implemented for the 2024 iteration of the Master's level "Robotics & XR" course taught by 
 [Dr. Ilkka Jormanainen](https://www.linkedin.com/in/ilkka-jormanainen-5954441/) at the [University of Eastern Finland](https://www.uef.fi/en), in collaboration with [Henki Robotics](https://henkirobotics.com/).
 
 [<img src="images/uef_logo.jpg" alt="UEF Logo" width="200" height="190"/>](https://www.uef.fi/en)
 [<img src="images/henki_robotics_logo.png" alt="Henki Robotics Logo" width="200" height="190"/>](https://henkirobotics.com/)
 
+## Course overview
+
+This course is designed to give you hands-on experience with the basics of robotics using ROS 2 and Gazebo simulation. 
+The exercises focus on the [Andino robot](https://github.com/Ekumen-OS/andino_gz/tree/humble) from Ekumen and are structured to gradually introduce you to ROS 2 and Docker.
+
+
+No prior experience in ROS 2 or Docker is needed, and since everything runs through Docker, you won’t need to install ROS 2 beforehand. 
```

Comment:
```suggestion
No prior experience with ROS 2 or Docker is needed, and since everything runs through Docker, you won’t need to install ROS 2 on your system beforehand. 
```